server/middleware/populateCurrentUser.ts

```diff
@@ -7,11 +8,16 @@ export default function populateCurrentUser(userService: UserService): RequestHa
     try {
       const { user } = req.session
       req.session.user = await userService.getUser(res.locals.user, user)
+
+      if (req.session.user.authSource !== 'nomis') {
+        return next(createHttpError.Forbidden())
+      }
+
```

Comment:
Not sure why we are checking the auth source here - has something changed to make us limit to only NOMIS users? Not that I think there are auth-type users logging in ... not yet anyway.

Reply:
Ok - makes sense. Yes - I've seen the same thing, and wondered what was happening. Probably best to stop it on entry rather than fail the caseload check later. Good call.